### DISCUSSION TITLE
Could you please provide the camera intrinsic and extrinsic parameters?

### DISCUSSION THREAD

**Discussion by software-ai-life** [bdd100k/bdd100k#163] [Q&A]
I've already download bdd100k images.
Could you please provide the camera intrinsic and extrinsic parameters?
If so, it's really helpful for my research.
Thank you very much!

**Comment by fyu** [comment]
The videos were captured by iPhone 5. You can estimate the intrinsics of the camera from the spec, but we don't have the calibration for each camera. You can get camera trajectories here: https://doc.bdd100k.com/usage.html#trajectories. Those are estimated by the iPhone 5 GPS and IMU sensors.
